src/FSharp.Control.TaskSeq/TaskSeq.fsi

```diff
@@ -871,6 +871,62 @@ type TaskSeq =
     /// <exception cref="T:ArgumentNullException">Thrown when the input task sequence is null.</exception>
     static member takeWhileInclusiveAsync: predicate: ('T -> #Task<bool>) -> source: TaskSeq<'T> -> TaskSeq<'T>
 
+    /// <summary>
+    /// Returns a task sequence that, when iterated, skips elements of the underlying sequence while the
+    /// given function <paramref name="predicate" /> returns <see cref="true" />, and then yields the remaining
+    /// elements. The first element where the predicate returns <see cref="false" /> is returned, which means that this
```

Comment:
skipWhile should not/does not return the first one that fails?
not getting the 0 or more elements

Does it sounds like I'm campaigning to remove the inclusive one to remove all this ambiguity yet?

Reply:
> skipWhile should not/does not return the first one that fails?

It says "it skips 0 or more elements", it does not _skip_ the first one that fails. Exactly like `Seq.skipWhile` does.

Reply:
> "The first element where the predicate returns false is returned,"

I was going to argue with this but I see now that it is technically correct. The point is that there is nothing special about the first item though - it just skips while it returns true. And then it returns the rest. Which includes the first item of the rest. And the rest of the rest. Including the very last item. if there is one.

>t says "it skips 0 or more elements", it does not skip the first one that fails.

You're probably correct - tbh I lost the will to parse it; it just feels very long winded, and I dont see whom explaining it in that manner actually helps...

----

Bottom line is that this level of over explanation is only justified if you're trying to explain it in terms of something that does have those factors in play.

i.e skipWhileInclusive.

----

TL;DR SkipWhile is really easy to explain if we just take the wording from Seq.takeWhile (which is not saddled with a contrived sibling :P)